book/installation/docker.md

```diff
@@ -125,3 +125,13 @@ docker exec -it reth bash
 **If Reth is running with Docker Compose, replace `reth` with `reth-reth-1` in the above command**
 
 Refer to the [CLI docs](../cli/cli.md) to interact with Reth once inside the Reth container.
+
+## Run only Grafana in Docker
+
+This allows importing existing Grafana dashboards, without running reth in Docker.
+
+```bash
+docker compose -f etc/docker-compose.yml up -d --no-deps grafana
+```
+
+After login with `admin:admin` credentials, this should list Prometheus under [`Grafana datasources`](http://localhost:3000/connections/datasources). Replace its `Prometheus server URL` with `http://host.docker.internal:9090`.
```

Comment:
Isn't this necessary regardless if reth is run in docker or not @shekhirin ? if so, it should be moved up a few LOC, just before 'Interacting with Reth inside Docker'

Reply:
it depends on how the Prometheus is set up. If Reth + Prometheus + Grafana is set up fully with `docker-compose.yml` that we provide, Grafana should connect to `prometheus:9090`